cmd/ef_tests/levm/runner.rs

```diff
@@ -242,6 +242,65 @@ fn get_post_value(test: &EFTest, tx_id: usize) -> Option<EFTestPostValue> {
     }
 }
 
+// Exceptions not covered: RlpInvalidValue and Type3TxPreFork
+fn exception_is_expected(
+    expected_exceptions: Vec<TransactionExpectedException>,
+    returned_error: VMError,
+) -> bool {
+    for expected_exception in expected_exceptions {
+        if matches!(
+            (expected_exception, &returned_error),
+            (
+                TransactionExpectedException::IntrinsicGasTooLow,
+                VMError::IntrinsicGasTooLow
+            ) | (
+                TransactionExpectedException::InsufficientAccountFunds,
+                VMError::InsufficientAccountFunds
+            ) | (
+                TransactionExpectedException::PriorityGreaterThanMaxFeePerGas,
+                VMError::PriorityGreaterThanMaxFeePerGas
+            ) | (
+                TransactionExpectedException::GasLimitPriceProductOverflow,
+                VMError::GasLimitPriceProductOverflow
+            ) | (
+                TransactionExpectedException::SenderNotEoa,
+                VMError::SenderNotEOA
+            ) | (
+                TransactionExpectedException::InsufficientMaxFeePerGas,
+                VMError::InsufficientMaxFeePerGas
+            ) | (
+                TransactionExpectedException::NonceIsMax,
+                VMError::NonceIsMax
+            ) | (
+                TransactionExpectedException::GasAllowanceExceeded,
+                VMError::GasAllowanceExceeded
+            ) | (
+                TransactionExpectedException::Type3TxBlobCountExceeded,
+                VMError::Type3TxBlobCountExceeded
+            ) | (
+                TransactionExpectedException::Type3TxZeroBlobs,
+                VMError::Type3TxZeroBlobs
+            ) | (
+                TransactionExpectedException::Type3TxContractCreation,
+                VMError::Type3TxContractCreation
+            ) | (
+                TransactionExpectedException::Type3TxInvalidBlobVersionedHash,
+                VMError::Type3TxInvalidBlobVersionedHash
+            ) | (
+                TransactionExpectedException::InsufficientMaxFeePerBlobGas,
+                VMError::InsufficientMaxFeePerBlobGas
+            ) | (
+                TransactionExpectedException::InitcodeSizeExceeded,
+                VMError::InitcodeSizeExceeded
+            )
+        ) {
+            return true;
+        }
+    }
+
+    false
+}
```

Comment:
We can use `any` here
```suggestion
// Exceptions not covered: RlpInvalidValue and Type3TxPreFork
fn exception_is_expected(
    expected_exceptions: Vec<TransactionExpectedException>,
    returned_error: VMError,
) -> bool {
    expected_exceptions.iter().any(|exception| {
        matches!(
            (exception, &returned_error),
            (
                TransactionExpectedException::IntrinsicGasTooLow,
                VMError::IntrinsicGasTooLow
            ) | (
                TransactionExpectedException::InsufficientAccountFunds,
                VMError::InsufficientAccountFunds
            ) | (
                TransactionExpectedException::PriorityGreaterThanMaxFeePerGas,
                VMError::PriorityGreaterThanMaxFeePerGas
            ) | (
                TransactionExpectedException::GasLimitPriceProductOverflow,
                VMError::GasLimitPriceProductOverflow
            ) | (
                TransactionExpectedException::SenderNotEoa,
                VMError::SenderNotEOA
            ) | (
                TransactionExpectedException::InsufficientMaxFeePerGas,
                VMError::InsufficientMaxFeePerGas
            ) | (
                TransactionExpectedException::NonceIsMax,
                VMError::NonceIsMax
            ) | (
                TransactionExpectedException::GasAllowanceExceeded,
                VMError::GasAllowanceExceeded
            ) | (
                TransactionExpectedException::Type3TxBlobCountExceeded,
                VMError::Type3TxBlobCountExceeded
            ) | (
                TransactionExpectedException::Type3TxZeroBlobs,
                VMError::Type3TxZeroBlobs
            ) | (
                TransactionExpectedException::Type3TxContractCreation,
                VMError::Type3TxContractCreation
            ) | (
                TransactionExpectedException::Type3TxInvalidBlobVersionedHash,
                VMError::Type3TxInvalidBlobVersionedHash
            ) | (
                TransactionExpectedException::InsufficientMaxFeePerBlobGas,
                VMError::InsufficientMaxFeePerBlobGas
            ) | (
                TransactionExpectedException::InitcodeSizeExceeded,
                VMError::InitcodeSizeExceeded
            )
        )
    })
}
```

Reply:
Done in [Use iter method 'any' instead of for](https://github.com/lambdaclass/ethrex/pull/1225/commits/8ffc5094e5b1eae9593db8e3b90b0aba1a11cccb)